core/test/results/sample_v2.json

```diff
@@ -7698,12 +7698,6 @@
         "duration": 100,
         "entryType": "measure"
       },
-      {
-        "startTime": 0,
-        "name": "lh:computed:EntityClassification",
-        "duration": 100,
```

Comment:
this kinda suggests that the computed audit isnt being run....

seems odd.

Reply:
It was being run multiple times no? This PR removes the unnecessary run.